graql/reasoner/tree/ResolutionTree.java

```diff
@@ -105,6 +105,7 @@ public void outputToFile() {
         Path profilePath = logPath.isAbsolute()? logPath : homePath.resolve(logPath);
 
         try {
+            profilePath.getParent().toFile().mkdirs();
```

Comment:
what's this ;p?

Reply:
it wasn't writing sometimes during tests because some of the path directory didn't exist - this makes them. Should be unnecssary outside of tests

Reply:
👍